core/src/components/toc-item/toc-item.tsx

```diff
@@ -78,6 +78,7 @@ export class TocItem {
   render() {
     const isParent = !!this.item.children?.length;
     const showItem = !this.filteredItems || this.filteredItems.has(this.item);
+    const url = this.item.url || `#${this.item.id || ''}`;
```

Comment:
should the `#${ before `this.item.id` come out?

Reply:
The old mechanism for determining the URL for a TOC item was to use the id attribute to create a link like `#chp_2`. We're now adding support for an optional `url` attribute, and falling back to the old ID if it's not there. The `#${...}\` is a javascript interpolation string equivalent to `f'#{...}'` in python. The `#` is literally just that, and the `$` indicates the interpolation inside the curlies.

Reply:
oh my bad, I missed the closing `}` 🤦